conf/base.config

```diff
@@ -64,4 +61,14 @@ process {
     withName:CUSTOM_DUMPSOFTWAREVERSIONS {
         cache = false
     }
+
+    // specific tool configs
+
+    withName:METASPADES {
+        cpus = {check_metaspades_cpus (10, task.attempt)}
+        memory = {check_max (64.GB * (2**(task.attempt-1)), 'memory')}
```

Comment:
is there anyway with nextflow to ask it to provision more resources if the first run fails? I know how to do this in snakemake but not in nextflow

Reply:
yes that is what this is doing, if it encounters an out of memory error it retires and scales up the memory by the task reattempt number